tests/integration/test_spans.py

```diff
@@ -1718,9 +1718,8 @@ def summarize_outcomes():
 
 
 @pytest.mark.parametrize(
-    "category,raises_rate_limited",
+    "category,hits_fast_path",
```

Comment:
I prefer the `raises_rate_limit`.

Reply:
Yeah, will change it back. I have changed it because of a test refactor but I rolled it back.